packages/angular/projects/angular/src/useReactive.ts

```diff
@@ -3,16 +3,33 @@
  * Copyright (c) 2022-present NAVER Corp.
  * MIT license
  */
-import { EventEmitter } from "@angular/core";
-import { ReactiveSubscribe, ReactiveAdapterParam, camelize, adaptReactive, ReactiveState } from "@cfcs/core";
+import { EventEmitter, NgZone } from '@angular/core';
+import {
+  ReactiveSubscribe,
+  ReactiveAdapterParam,
+  camelize,
+  adaptReactive,
+  ReactiveState,
+} from '@cfcs/core';
 
 export function useReactive<
   Instance extends ReactiveSubscribe<Record<string, any>>,
   State extends Record<string, any> = ReactiveState<Instance>,
   Methods extends keyof Partial<Instance> = any,
   Props = any,
-  Events extends Record<string, any> = {},
->(self: State, reactiveAdapter: ReactiveAdapterParam<Instance, State, Methods, Props, Events>, props?: () => Props) {
+  Events extends Record<string, any> = {}
+>(
+  self: State,
+  reactiveAdapter: ReactiveAdapterParam<
+    Instance,
+    State,
+    Methods,
+    Props,
+    Events
+  >,
+  ngZone: NgZone,
```

Comment:
We're not quite ready for angular yet.

Preparing for angular internally.
I hope the code to conditionally handle ngZone is added.

Reply:
I've marked it as optional.